AstronomyUITests/AstronomyUITests.swift

```diff
@@ -0,0 +1,73 @@
+//
+//  AstronomyUITests.swift
+//  AstronomyUITests
+//
+//  Created by Rob Vance on 9/24/20.
+//  Copyright © 2020 Lambda School. All rights reserved.
+//
+
+import XCTest
+@testable import Astronomy
+
+class AstronomyUITests: XCTestCase {
+    
+    var app: XCUIApplication {
+        return XCUIApplication()
+    }
+    
+    private var saveButton: XCUIElement {
+        return app.buttons["PhotoDetailViewController.SaveButton"]
+    }
+    
+    func testSavingPhoto() {
+        // Provides an opportunity to reset the state and to throw errors before call each test method
+        try! setUpWithError()
+        
+        let app = XCUIApplication()
+        
+        app.launch()
+        app.collectionViews.children(matching:.cell).element(boundBy:0).otherElements.containing(.image,identifier:"imageCell").element.tap()
+        saveButton.tap()
+        let photoSavedAlert = app.alerts["Photo Saved!"].scrollViews.otherElements.buttons["Okay"]
+        
+        XCTAssertEqual(photoSavedAlert.label, "Okay")
+        
+    }
+    
+    func testChangingSols() {
+        try! setUpWithError()
+        
+        app.launch()
+        let app = XCUIApplication()
+        app.navigationBars["Sol 1"]/*@START_MENU_TOKEN@*/.buttons["PhotosCollectionViewController.NextSolButton"]/*[[".buttons[\">\"]",".buttons[\"PhotosCollectionViewController.NextSolButton\"]"],[[[-1,1],[-1,0]]],[0]]@END_MENU_TOKEN@*/.tap()
+        app.navigationBars["Sol 2"]/*@START_MENU_TOKEN@*/.buttons["PhotosCollectionViewController.NextSolButton"]/*[[".buttons[\">\"]",".buttons[\"PhotosCollectionViewController.NextSolButton\"]"],[[[-1,1],[-1,0]]],[0]]@END_MENU_TOKEN@*/.tap()
+        app.navigationBars["Sol 3"]/*@START_MENU_TOKEN@*/.buttons["PhotosCollectionViewController.NextSolButton"]/*[[".buttons[\">\"]",".buttons[\"PhotosCollectionViewController.NextSolButton\"]"],[[[-1,1],[-1,0]]],[0]]@END_MENU_TOKEN@*/.tap()
```

Comment:
Your tests would read a bit clearer if you used variables to fetch the buttons.

``` swift
func testMultiStepNavEndsOnSol16() throws {
    previousSolButton.tap()
    firstCell.tap()
    navBarLeftButton.tap()
    nextSolButton.tap()
    nextSolButton.tap()
    firstCell.tap()
    navBarLeftButton.tap()
    XCTAssertEqual(navBarTitleLabel, "Sol 16")
}
```

Also, try to name your tests to include the expected outcome in the name of the test.